Resources/public/js/views/dashboard/ez-dashboardblockallcontentview.js

```diff
@@ -12,130 +12,49 @@ YUI.add('ez-dashboardblockallcontentview', function (Y) {
-         *
-         * @method render
-         * @return {eZ.DashboardBlockAllContentView} the view itself
-         */
-        render: function () {
-            var items = this.get('items').map(function (item) {
-                    return {
-                        /*
-                         * @TODO remove content model
-                         * see  https://jira.ez.no/browse/EZP-25842
-                         */
-                        content: item.content.toJSON(),
-                        contentType: item.contentType.toJSON(),
-                        location: item.location.toJSON(),
-                        contentInfo: item.location.get('contentInfo').toJSON(),
-                    };
-                });
-
-            this.get('container').setHTML(this.template({
-                items: items,
-                loadingError: this.get('loadingError'),
-            }));
-
-            return this;
-        },
-
-        /**
-         * Shows the row options
-         *
-         * @method _uiShowOptions
-         * @protected
-         * @param event {Object} event facade
-         */
-        _uiShowRowOptions: function (event) {
-            this._uiHideRowOptions();
-
-            event.currentTarget.addClass(CLASS_ROW_SELECTED);
-
-            this._clickOutsideHandler = this.get('container').one(SELECTOR_CONTENT).on('clickoutside', Y.bind(this._uiHideRowOptions, this));
         },
 
         /**
-         * Clears the click outside handler
+         * Gets item template data.
          *
-         * @method _clearClickOutsideHandler
+         * @method _getTemplateItem
          * @protected
+         * @param item {Object} item data
          */
-        _clearClickOutsideHandler: function () {
-            if (this._clickOutsideHandler) {
-                this._clickOutsideHandler.detach();
-            }
-        },
-
-        /**
-         * Hides the row options
-         *
-         * @method _uiHideOptions
-         * @protected
-         */
-        _uiHideRowOptions: function () {
-            this.get('container').all(SELECTOR_ROW).removeClass(CLASS_ROW_SELECTED);
-
-            this._clearClickOutsideHandler();
-        },
-
-        /**
-         * Removes the loading state of the UI
-         *
-         * @method _uiEndLoading
-         * @protected
-         */
-        _uiEndLoading: function () {
-            this.get('container').removeClass(CLASS_LOADING);
+        _getTemplateItem: function (item) {
+            return {
+                /*
+                 * @TODO remove content model
+                 * see  https://jira.ez.no/browse/EZP-25842
+                 */
+                content: item.content.toJSON(),
+                contentType: item.contentType.toJSON(),
+                location: item.location.toJSON(),
+                contentInfo: item.location.get('contentInfo').toJSON(),
+            };
         },
 
         /**
          * Makes request for data
```

Comment:
this should be adjusted to explain that in this implementation, the location search event is fired.